src/main/resources/application.yaml

```diff
@@ -29,3 +29,9 @@ site-mappings:
       poBox: 12746
       siteId: ${SITE_ID_FINREM}
 
+azure:
+  servicebus:
+    payments:
+      connection-string: ${PAYMENTS_QUEUE_READ_CONNECTION_STRING}
```

Comment:
is this ever set?

Reply:
that is set through bootstrap for AAT and PROD. For preview it is done through using preview template yaml

Reply:
In the chart it is. But yea - not sure about other environments